jina/optimizers/parameters.py

```diff
@@ -19,6 +21,11 @@ def __init__(
 
 
 class IntegerParameter(OptimizationParameter):
+    """Used for optimizing integer parameters with the Jina Optimizer.
+       For a detailed information about sampling and usage see
```

Comment:
information is always plural.
```suggestion
       For detailed information about sampling and usage see
```